tests/unit/test_fileio.py

```diff
@@ -388,7 +388,7 @@ def test_seek(self):
         with self.assertRaises(ValueError):
             pos = buff.tell()
             buff.seek(len(TEST_BINARY_DATA) + 1)
-            self.assertEqual(pos, buff.tell())
+        self.assertEqual(pos, buff.tell())
```

Comment:
what happened here? is there more context

Reply:
Line 390 raises the exception in which the destination byte is not contained in buffer, so the test never hits Line 391. This change ensures all assertions are run.